sylvia-derive/src/associated_types.rs

```diff
@@ -5,7 +5,8 @@ use syn::{
     WhereClause, WherePredicate,
 };
 
-const RESERVED_TYPES: [&str; 3] = ["Error", "QueryC", "ExecC"];
+const ERROR_TYPE: &str = "Error";
+const RESERVED_TYPES: [&str; 3] = [ERROR_TYPE, "QueryC", "ExecC"];
```

Comment:
If there are some reserved type it would be good to describe them in README, examples and at the end in the Sylvia book as well.  It is also relevant whether they are optional or mandatory.